src/qtmozembed/qtmozembed.pri

```diff
@@ -0,0 +1,18 @@
+INCLUDEPATH += $$PWD
```

Comment:
Maybe this should be called something like "engine" so that it would not be tightly coupled to the qtmozembed. We clearly need to isolate this one from the "core".